docs/actors/development/builds.md

```diff
@@ -13,32 +13,32 @@ paths:
 
 Before the actor can be run, it first needs to be built. The build effectively creates a snapshot of a specific version of the actor's settings such as the [Source code]({{@link actors/development/source_code.md}}) and [Environment variables]({{@link actors/development/environment_variables.md}}), and creates a Docker image that contains everything the actor needs for its run, including necessary NPM packages, web browsers, etc.
 
-Each build is assigned a unique build number of the form `MAJOR.MINOR.BUILD` (e.g. `1.2.345`), where `MAJOR.MINOR` corresponds to the actor version number (see [Versioning]({{@link actors/development/source_code.md#versioning}})) and `BUILD` is an automatically-incremented number starting at `1`.
+Each build is assigned a unique build number of the form *MAJOR.MINOR.BUILD* (e.g. *1.2.345*), where *MAJOR.MINOR* corresponds to the actor version number (see [Versioning]({{@link actors/development/source_code.md#versioning}})) and *BUILD* is an automatically-incremented number starting at *1*.
```

Comment:
Pretty strange - this line has unnecessary * symbols when built: docs.apify.com/actors/development/builds?version=develop in second paragraph